src/main/webapp/assets/modules/common/utils.fac.js

```diff
@@ -4,7 +4,7 @@ angular.module("Common").factory("utils", function (jsonValue, $location, $rootS
   var instance = {
     openFBShare: function(uri) {
       window.open(
-        'https://www.facebook.com/sharer/sharer.php?u=' + baseUrl + uri,
+        'https://www.facebook.com/sharer/sharer.php?u=' + baseUrl + uri + '&redirect_uri=' + baseUrl + uri,
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'baseUrl' is not defined.